### DISCUSSION TITLE
Whether the minio cluster can delete disk files directly？Will it make the cluster unavailable？

### DISCUSSION THREAD

**Discussion by PanErheng** [minio/minio#16327] [Q&A] [answered]
My cluster now stores about 10 million pictures, each about 700 KB or 800 KB. Now I want to delete them, but I use the mc command to delete them too slowly. Therefore, I want to delete disk files at all nodes of the cluster at the same time, but I don't know whether it will cause other problems

environment：
minio version:minio:RELEASE.2022-05-26T05-48-41Z
Docker deploys minio cluster with 4 nodes,
32G memory and 48 core cpu per node and disk is a 5T HDD,

**Comment by harshavardhana** [comment, marked as answer]
Deleting from the backend directly is undefined @PanErheng - use it at your own risk we do not support it.

**Reply by PanErheng** [reply]
OK!!! Thank you for your reply
